Alignment/OfflineValidation/macros/FitPVResiduals.C

```diff
@@ -43,7 +43,7 @@
 #include <TSystem.h>
 #include <TTimeStamp.h>
 #include <TStopwatch.h>
-//#include "Alignment/OfflineValidation/plugins/TkAlStyle.cc" 
+//#include "Alignment/OfflineValidation/macros/TkAlStyle.cc" 
```

Comment:
as you moved the `TkAlStyle.cc` from plugins to macros I am wondering if we could also dump the dependency on:
```
<use   name="rootgraphics"/>
<use   name="rootinteractive"/>
```
we have in https://github.com/cms-sw/cmssw/blob/master/Alignment/OfflineValidation/plugins/BuildFile.xml#L52.
This is a dependency violation and moreover is effectively barring us from submitting condor jobs.

Reply:
Right, I remember you mentioned that on mattermost.  Done.